x-pack/plugins/cases/public/components/user_actions/common.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiCommentProps, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+
+import { Actions, ConnectorUserAction } from '../../../common/api';
+import { UserActionTimestamp } from './timestamp';
+import { UserActionResponse } from './types';
+import { UserActionUsernameWithAvatar } from './avatar_username';
+import { UserActionCopyLink } from './copy_link';
+import { UserActionMoveToReference } from './move_to_reference';
+import { CaseUserActions } from '../../containers/types';
+
+interface Props {
+  userAction: UserActionResponse<ConnectorUserAction>;
+  handleOutlineComment: (id: string) => void;
+}
+
+const CommentListActions: React.FC<Props> = React.memo(({ userAction, handleOutlineComment }) => (
+  <EuiFlexGroup responsive={false}>
+    <EuiFlexItem grow={false}>
+      <UserActionCopyLink id={userAction.actionId} />
+    </EuiFlexItem>
+    {userAction.action === Actions.update && userAction.commentId != null && (
```

Comment:
nit: looks like we use this if check a few places might be worth moving it to its own function

Reply:
I could not find any place with the same check. Could you please point out the files?

Reply:
Sorry, I should have included that in my first comment, I think the same check is at the bottom of this file right?

https://github.com/elastic/kibana/blob/0d70da849c9ca92747f31409ee03db3d1a29e956/x-pack/plugins/cases/public/components/user_actions/common.tsx#L70